apps/dashboard/app/javascript/packs/path_selector/sweet_alert.js

```diff
@@ -0,0 +1,91 @@
+import Swal from 'sweetalert2'
+import {CONTENTID, EVENTNAME as DATATABLE_EVENTNAME} from './data_table.js';

```

Comment:
This is why we can't reuse the other `sweet_alert.js`, we have this dependency on `data_table.js`.

Reply:
Thanks Jeff!